.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,38 @@
+name: Benchmark
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Not specific to this PR, but should we run CI/benchmark on pushes to master as well?

Reply:
I can add CI to master mergers - I was hesitant at first because the CI here actually takes a long time. 

Not sure it is worth benchmarking the master merge as well.